Initialize/init.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+set -e 
+trap "exit" INT
+
+function print_usage {
+    cat <<EOF
+Command 
+    $(basename $0)
+Arguments
+   --prefix|-p                          [Requied] Used to distingush your perf resources with others'
+   --subscription|-s                    [Requied] The subscriton used to create resources
+   --location|-l                        [Requied] The location used to create resouces
+   --help|-h                            Print help
+EOF
+}
+
+function throw_if_empty {
+    local name="$1"
+    local value="$2"
+    if [[ -z "$value" ]];then
+        echo "ERROR: Parameter $name cannot be empty." 1>&2
+        exit -1
+    fi
+}
+
+while [[ "$#" > 0 ]];do
+    key="$1"
+    shift
+    case $key in 
+        --prefix|-p)
+            PREFIX="$1"
+            shift
+        ;;
+        --subscription|-s)
+            SUBSCTIPTION="$1"
+            shift
+        ;;
+        --location|-l)
+            LOCATION="$1"
+            shift
+        ;;
+        --help|h)
+            print_usage
+            exit
+        ;;
+        *)
+            echo "ERROR: Unknow argument '$key'" 1>&2
+            print_usage
+            exit -1
+    esac
+done
+throw_if_empty "subscription" $subscription
+throw_if_empty "prefix" $PREFIX
+throw_if_empty "location" $LOCATION    
+
+az account set -s $subscription
```

Comment:
add `az cloud set --name xxx` for Global/Mooncake/FireFax...?

Reply:
add cloud as optional